.gitignore

```diff
@@ -76,7 +76,7 @@ target/
 **/.vscode/*
 
 # Hosted assets
-experimenter/experimenter/static/scripts/
+experimenter/experimenter/static/scripts/bundle.js
```

Comment:
why this change?

Reply:
Oh , this was for the modal script I'll update it